ocp_resources/resource.py

```diff
@@ -265,33 +264,31 @@ def get_client(
             temp_file_path=temp_file_path,
         )
     else:
-        # Ref: https://github.com/kubernetes-client/python/blob/v26.1.0/kubernetes/base/config/__init__.py
-        LOGGER.info("Trying to get client via new_client_from_config")
-
-        # kubernetes.config.kube_config.load_kube_config sets KUBE_CONFIG_DEFAULT_LOCATION during module import.
-        # If `KUBECONFIG` environment variable is set via code, the `KUBE_CONFIG_DEFAULT_LOCATION` will be None since
-        # is populated during import which comes before setting the variable in code.
-        config_file = config_file or os.environ.get("KUBECONFIG", "~/.kube/config")
-
-        _client = kubernetes.config.new_client_from_config(
-            config_file=config_file,
-            context=context,
-            client_configuration=client_configuration,
-            persist_config=persist_config,
-        )
+        try:
```

Comment:
the code should reflect my comment in line 214, only if none of them passed we should do client from incluster config